src/routes/authentication.ts

```diff
@@ -30,28 +38,37 @@ export default async function authentication(fastify: FastifyInstance) {
     const data: any = req.body;
     const { username, password, domain } = data;
 
-    const authInfo = Config.getAuthenticationInfo(domain);
-    let chtSession;
+    const getLoginErrorMessage = (error: unknown): string => {
+      if (error instanceof AuthError) {
+        console.error('Login error:', error.message);
+        return error.errorMessage;
+      }
+
+      console.error('Login error:', error);
```

Comment:
can we log once after the call to `getLoginErrorMessage` to avoid multiple log statements?